### PR TITLE
Modified to show the FiveM player list

### PR DESCRIPTION
**Pull request by Erani0**
A small change to the script to get the FiveM player list and display it with

### CODE REVIEW

**Comment by Erani0** [comment]
Maybe one of you knows how to fix this possible error
IssueId: 48744890  Message: It seems like ``@file_get_contents(sprintf('http://%s/players.json', $connection))`` can also be of type ``false``. However, the property ``$PlayerList`` is declared as type ``string``. Maybe add an additional type check?  Filename: src/GameQ/Protocols/Gta5m.php  LineNumber: 117  Link: https://scrutinizer-ci.com/g/Austinb/GameQ/inspections/b7af9916-09d2-4020-b4b3-9b25d38a2f8f/issues/files/src/GameQ/Protocols/Gta5m.php?status=new&orderField=path&order=asc&honorSelectedPaths=0&issueId=48744890

**Comment by Erani0** [comment]
Sorry about the force push thought it was better than dozens of commits.
I'm really clueless why scrutinizer always outputs an error does anyone know how to fix it so that there are no problems

**Comment by Erani0** [comment]
I give up scrutinizer always says there is an error I can not find a solution

**Comment by Erani0** [comment]
Finally scrutinizer has no more errors.

File: src/GameQ/Protocols/Gta5m.php [hunk 1]
```diff
@@ -163,6 +185,15 @@ protected function processStatus(Buffer $buffer)
...
+        if ($this->PlayerList) {
+            $players = [];
+            $data = json_decode($this->PlayerList);
+            foreach ($data as $player) {
+                $players[] = array("id"=>$player->id,"name"=>$player->name,"ping"=>$player->ping,"identifiers"=>$player->identifiers);
+            }
+            $result->add('players', $players);
+        }
+
         /*var_dump($data);
```
Comment (Austinb) [inline src/GameQ/Protocols/Gta5m.php]: Remove commented code used when building/testing.

**Comment by Austinb** [comment]
Please remove any comments that are not needed (i.e. preventing test code from operating).  

Also, as a bigger discussion, this library was built to only use the bare minimum of requirements due to hosting and other restrictions.  That being said I know some hosts will block functions to prevent abuse and other issues.  The question is should an internal call to \Protocols\Http (https://github.com/Austinb/GameQ/blob/v3/src/GameQ/Protocols/Http.php) be used instead of making direct calls to `file_get_contents()`?  This is just a thought since this solution is a shoehorn option given that the protocol this game server is using does not return the player information in a "regular" query call. 

Thoughts?
CC: @bumbummen99

File: src/GameQ/Protocols/Gta5m.php (hunk 1 above)
Comment (Erani0) [reply src/GameQ/Protocols/Gta5m.php]: you mean this? 
/*var_dump($data);
If so this is not from me this was already before my editing in the protocol

**Comment by Erani0** [comment]
So the comments were already there before but it can also be removed.
And yes I understand what you mean.
But FiveM outputs data differently, you can query them via http then you would have to query 2 urls and evaluate or as it is now and subsequently integrate the player list as I have done now.

**Comment by bumbummen99** [comment]
> Please remove any comments that are not needed (i.e. preventing test code from operating).
> 
> Also, as a bigger discussion, this library was built to only use the bare minimum of requirements due to hosting and other restrictions. That being said I know some hosts will block functions to prevent abuse and other issues. The question is should an internal call to \Protocols\Http (https://github.com/Austinb/GameQ/blob/v3/src/GameQ/Protocols/Http.php) be used instead of making direct calls to `file_get_contents()`? This is just a thought since this solution is a shoehorn option given that the protocol this game server is using does not return the player information in a "regular" query call.
> 
> Thoughts? CC: @bumbummen99

How would that internal call look like?

**Comment by Erani0** [comment]
You get this information from the server.
http://45.131.108.130:30120/info.json

**Comment by Austinb** [comment]
Should PHP 5.6 be removed from the checks?  It may already have been.  @bumbummen99 can you bump this so it will check again?

**Comment by bumbummen99** [comment]
@Austinb It failed due to the null coalesce operator (`??`) that was introduced with PHP7. While i agree that we should drop support for ancient versions to encurage people to update their code and environment to a secure and supported version, i would say that this should be done with a seperate change / release so that this can be clearly communicated - we could then also introduce a **lot** of new PHP features in regards to types that i think would come in very handy.
Edit: Also think this PR is good now :)

**Comment by TrustHeberg** [comment]
Query for FiveM , change.. A update coming ?

**Pull request merged**